sessionctx/variable/sysvar.go

```diff
@@ -741,6 +750,7 @@ var defaultSysVars = []*SysVar{
 	{Scope: ScopeGlobal | ScopeSession, Name: TiDBAnalyzeVersion, Value: strconv.Itoa(DefTiDBAnalyzeVersion), Type: TypeInt, MinValue: 1, MaxValue: 2},
 	{Scope: ScopeGlobal | ScopeSession, Name: TiDBEnableIndexMergeJoin, Value: BoolToOnOff(DefTiDBEnableIndexMergeJoin), Type: TypeBool},
 	{Scope: ScopeGlobal | ScopeSession, Name: TiDBTrackAggregateMemoryUsage, Value: BoolToOnOff(DefTiDBTrackAggregateMemoryUsage), Type: TypeBool},
+	{Scope: ScopeGlobal | ScopeSession, Name: TiDBMultiStatementMode, Value: Warn, Type: TypeEnum, PossibleValues: []string{Off, On, Warn}},
```

Comment:
If a new global variable is added,  add it to here https://github.com/pingcap/tidb/blob/030a00e55d04e58ca615794f835bef144eec3083/session/session.go#L2476

And please take special care of the default value: as the data is persisted, we have to consider that for both a new bootstraped cluster and a upgraded cluster.

Reply:
It is there :-) See:
https://github.com/pingcap/tidb/pull/22351/files#diff-10dab83c04a99358d431906d37f48b92be97d50a2719d4ddf81c62d3b03a0aebL2476-L2477

The default value is `WARN`, and the switch statement handles this as the default case. In future, we will change the default, and I will write a bootstrap task to force change it for users upgrading.